spec/api-web-contents-spec.js

```diff
@@ -754,4 +754,23 @@ describe('webContents module', () => {
       })
     })
   })
+
+  describe('webframe messages in sandboxed contents', () => {
+    it('responds to executeJavaScript', (done) => {
+      w.destroy()
```

Comment:
This should not be needed, there is already a cleanup of open windows after each spec.